velox/connectors/hive/storage_adapters/gcs/GCSFileSystem.cpp

```diff
@@ -275,6 +276,25 @@ class GCSFileSystem::Impl {
     }
     options.set<gcs::UploadBufferSizeOption>(kUploadBufferSize);
 
+    auto max_retry_count = hiveConfig_->gcsMaxRetryCount();
+    if (max_retry_count) {
+      LOG(INFO) << "Config::gcsMaxRetryCount = " << max_retry_count.value();
```

Comment:
We don't log other config values (even for other FileSystems). Can we remove this logging here for now?
Same below.

Reply:
Ouch! My mistake! I will fix it.